Formula/shibboleth-idp.rb

```diff
@@ -0,0 +1,23 @@
+class ShibbolethIdp < Formula
+  desc "SAML Identity Provider implementing Single Sign-On functionality"
+  homepage "https://www.shibboleth.net/"
+  url "https://shibboleth.net/downloads/identity-provider/3.3.2/shibboleth-identity-provider-3.3.2.tar.gz"
+  sha256 "ed9fbefd273199d2841d4045b2661671c53825ed3c7d52d38bfe516b39d5fc64"
+
+  bottle :unneeded
+
+  depends_on :java => "1.8+"
+
+  def install
+    # Remove Windows scripts
+    rm_rf Dir["bin/*.bat"]
+
+    # Install files
+    libexec.install Dir["*"]
+  end
+
+  test do
```

Comment:
We require new formulae in homebrew-core to have a more substantive test than just `version` or `help`.